FieldOpt/CMakeLists.txt

```diff
@@ -62,7 +88,7 @@ if (NOT BUILD_WIC_ONLY)
     # Copy execution scripts
     file(GLOB EXECUTION_SCRIPTS
             "Simulation/execution_scripts/*.sh")
-    file(COPY ${EXECUTION_SCRIPTS} DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/execution_scripts)
+    file(COPY ${EXECUTION_SCRIPTS} DESTINATION ${CMAKE_BINARY_DIR}/execution_scripts)
```

Comment:
I experienced some issues with the former, I think things where not copying for some reason... If they are equivalent, then we can keep CMAKE_BINARY_DIR since it is also used in the beginning.